### DISCUSSION TITLE
Custom field sizes?

### DISCUSSION THREAD

**Discussion by rob230580** [Courseplay/Courseplay_FS22#2482] [General]
I cant find a mod to calculate custom field sizes (Mod map Hobo's doesnt show them), so is there a way for courseplay to do it? The field generator is automatic where as i seem to remember cp gave a field size in fs19....

Also is anyone else working without AD? I find the AI on looping works really well with any form of chaser for harvesting?

**Comment by Tensuko** [comment]
Field size is shown on the AI Map if you click on the field number. For custom fields, draw a field border (or record it) and then click on the name to get the size.

I dont understand your 2nd question.

**Reply by rob230580** [reply]
Ah I cant believe I didn't know that. Thank you 

My other comment was just an observation that I find you don't really need Auto-drive apart from Bunker filling, I just use courseplay with the AI looping.

Keep up the great work :)
